chykehyman/Grocerrific
# ch(test): write api tests

### What does this PR do?
Adds application's backend api tests

### Description of Task to be completed?

- [x] setup testing environment
- [x] add api test suites

### How should this be manually tested?
```
Clone this repository and cd into the it
Run `npm install` to install all needed packages
Then run `npm test`
Terminate the process using `ctrl c` to see the generated coverage
```


Any background context you want to provide?

Nil

### What are the relevant pivotal tracker stories?

Nil

### Screenshots (if appropriate)
<img width="692" alt="screenshot 2018-12-18 at 12 12 14 pm" src="https://user-images.githubusercontent.com/22524619/50151051-18873000-02c0-11e9-89fc-34cb4778202f.png">
